### DISCUSSION TITLE
Centering an image

### DISCUSSION THREAD

**Discussion by lordofscripts** [johnfercher/maroto#437] [Q&A]
I have a JPG image in a file and I want to accomplish these two use cases separately:

- Center the image on the current page, both horizontally and vertically.
- Center the image horizontally at the current page/row position.

Issues:

- How can I have the image automatically centered within the current column. i.e. `NewFromFileCol()`
- How to know how many columns I need for the image?
- How do I know if the height of the image is too much and would cause the image to overflow to the next page?

@johnfercher

**Comment by johnfercher** [comment]
You have to use the opcional props, see the example.
